frontend/src/Component/Share/Navbar/Sidebar.jsx

```
@@ -90,8 +92,19 @@ const Sidebar = ({ onLogout }) => {
                         )}
                     </>
                 )}
+
+                <button className="flex items-center mt-auto ">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)